File: Data Structures & Algorithms/find-all-numbers-disappeared-in-an-array/submission-0.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        Set<Integer> set = new HashSet<Integer>();
        for (int num : nums) set.add(num);
        List<Integer> result = new ArrayList<Integer>();
        for(int i=1; i <= nums.length; i++){
            if(!set.contains(i)){
                result.add(i);
            }
        }
        return result;
    }
}

File: Data Structures & Algorithms/special-array-i/submission-0.py
class Solution:
    def isArraySpecial(self, nums: List[int]) -> bool:
        prev = nums[0] % 2
        for i in range(1,len(nums)):
            if (nums[i] % 2 == 0 and prev == 0) or (nums[i] % 2 == 1 and prev == 1):
                return False
            prev = nums[i] % 2
        return True